### ISSUE TITLE
Auswertung von BirdNet, Perch [done]

### ISSUE THREAD

**Issue by lurauch** [DBD-research-group/BirdSet#175]

**Comment by lurauch** [comment]
BirdNET: tbd

**Comment by lurauch** [comment]
- perch finished and logits for NIPS4BPLUS finished
- BirdNET evaluation not so easy (3 second test samples would be necessary)

**Comment by lurauch** [comment]
- maybe birdclef two years ago? @reheinrich

**Comment by lurauch** [comment]
- 5 second chunk in 2 overlapping 3 second chunks
- für beide Vorhersage mit logits, maximum over logits for final prediction
